content/developers/api-reference/events-api/index.md

```diff
@@ -62,457 +48,63 @@ ASSET_UUID=<ASSET_UUID>
-      "arc_blob_hash_value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-      "arc_blob_identity": "blobs/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-      "arc_blob_hash_alg": "SHA256",
-      "arc_file_name": "safety_conformance.pdf",
-      "arc_display_name": "Conformance Report",
-    },
-    "arc_primary_image": {
-      "arc_attribute_type": "arc_attachment",
-      "arc_blob_hash_value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-      "arc_blob_identity": "blobs/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-      "arc_blob_hash_alg": "SHA256",
-      "arc_file_name": "photo.jpg",
-      "arc_display_name": "arc_primary_image",
-    },
-  },
-  "timestamp_declared": "2019-11-27T14:44:19Z",
-  "principal_declared": {
-    "issuer": "idp.synsation.io/1234",
-    "subject": "phil.b",
-    "email": "phil.b@synsation.io"
-  },
-}
-EOF
-```
-
-Add the request to the Asset Record by POSTing it to the resource:
-
-```bash
-curl -X POST \
-    -H "@$HOME/.datatrails/bearer-token.txt" \
-    -H "Content-type: application/json" \
-    -d "@/tmp/event.json" \
-    https://app.datatrails.ai/archivist/v2/assets/$ASSET_UUID/events
-```
-
-The response:
-
-```json
-{
-  "identity": "assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "asset_identity": "assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-  "operation": "Record",
-  "behaviour": "RecordEvidence",
-  "event_attributes": {
-    "arc_display_type": "Safety Conformance",
-    "arc_description": "Safety conformance approved for version 1.6. See attached conformance report",
-    "arc_evidence": "DVA Conformance Report attached",
-    "conformance_report": {
-      "arc_attribute_type": "arc_attachment",
-      "arc_blob_hash_value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-      "arc_blob_identity": "blobs/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-      "arc_blob_hash_alg": "SHA256",
-      "arc_file_name": "safety_conformance.pdf",
-      "arc_display_name": "Conformance Report",
-    },
-    "arc_primary_image": {
-      "arc_attribute_type": "arc_attachment",
-      "arc_blob_hash_value": "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-      "arc_blob_identity": "blobs/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-      "arc_blob_hash_alg": "SHA256",
-      "arc_file_name": "safety_conformance.pdf",
-      "arc_display_name": "Conformance Report",
-    },
-  },
-  "timestamp_accepted": "2019-11-27T15:13:21Z",
-  "timestamp_declared": "2019-11-27T14:44:19Z",
-  "timestamp_committed": "2019-11-27T15:15:02Z",
-  "principal_declared": {
-    "issuer": "idp.synsation.io/1234",
-    "subject": "phil.b",
-    "email": "phil.b@synsation.io"
-  },
-  "principal_accepted": {
-    "issuer": "job.idp.server/1234",
-    "subject": "bob@job"
-  },
-  "confirmation_status": "COMMITTED",
-  "block_number": 12,
-  "transaction_index": 5,
-  "transaction_id": "0x07569"
-}
-```
-
```

Comment:
I've removed the attachment docs from here as the flow is quite broken. The user must navigate away from the these docs, create an attachment, save the id and hash, then return here. 
As we're also not prioritizing attachments, I've streamlined the events docs to create and query events. 
We can/should have separate docs for the attachments workflow.

Reply:
That’s not valid. We need to keep attachments. If the flow doesn’t work then fix it, we can’t just delete it.

Reply:
How about if I move the attachements to a separate doc, even part of this PR, but not in the event doc flow?

Reply:
One place with the details and every other place with a reference is a fine model for sure. But it has to be clear how you include the attachment reference in an Event